### DISCUSSION TITLE
running dev builds of frigate

### DISCUSSION THREAD

**Discussion by JustinScott** [blakeblackshear/frigate#11637] [General]
Anyone mind explaining how dev branch images work? I noticed you can use the dev tag and commit hash, but it doesn't seem to work for all commit hashes. How can  I tell which commits have images or what the latest image is?

```
    image: ghcr.io/blakeblackshear/frigate:dev-cf4517c
```

**Comment by NickM-27** [comment]
Just check the packages section in the sidebar on GitHub to see the builds
